### DISCUSSION TITLE
Convert generic pytorch models (such as chatglm-6b) to ggml

### DISCUSSION THREAD

**Discussion by linouxis9** [ggml-org/llama.cpp#553] [Q&A] [answered]
Has the community built any scripts to convert generic pytorch models such as https://huggingface.co/THUDM/chatglm-6b to ggml for usage with llama.cpp? Thanks 😄

**Comment by iHaagcom** [comment]
Any news about these models?

**Comment by linouxis9** [comment, marked as answer]
@iHaagcom You could try https://github.com/akx/ggify which seem to be a tool that can do that

**Reply by iHaagcom** [reply]
> @iHaagcom You could try https://github.com/akx/ggify which seem to be a tool that can do that

Unfortunately for ChatGLM6B it returned the following error: returned non-zero exit status 1 exit status
